networks/rpc/server.go

```diff
@@ -49,6 +49,9 @@ const (
 
 	// concurrencyLimit is a limit for the number of concurrency connection for RPC servers
 	concurrencyLimit = 3000
+
+	// maxSubscriptionPerConn is a maximum number of subscription for a server connection
+	maxSubscriptionPerConn = 10
```

Comment:
I think this value should be parameterized.
This limits the number of subscription and a node operator might wants to subscribe more than 10 times.